src/importexport/mei/internal/meiimporter.cpp

```diff
@@ -2492,6 +2496,41 @@ bool MeiImporter::readFermata(pugi::xml_node fermataNode, Measure* measure)
     return true;
 }
 
+/**
+ * Read a fing.
+ */
+
+bool MeiImporter::readFing(pugi::xml_node fingNode, Measure* measure)
+{
+    IF_ASSERT_FAILED(measure) {
+        return false;
+    }
+
+    bool warning;
+    libmei::Fing meiFing;
+    meiFing.Read(fingNode);
+
+    Note* note = this->findStartNote(meiFing);
+    if (!note) {
+        // Here we could detect if it's a tied chord (for files not exported from MuseScore)
+        // We would need a dedicated list and tie each note once the second chord has been found.
+        return true;
+    }
```

Comment:
Is this part really relevant for fingerings? (looks like it was accidentally copied over from ties)

Reply:
Yes, forgot to clean this up.